sdk/state/integrationtests/helpers_test.go

```diff
@@ -17,7 +17,7 @@ import (
 
 // functions to be used in the state_test integration tests
 
-func initAccounts(t *testing.T, client horizonclient.ClientInterface, asset txnbuild.Asset, assetLimit string, distributorKP *keypair.Full) (initiator Participant, responder Participant) {
+func initAccounts(t *testing.T, asset txnbuild.Asset, assetLimit string, distributorKP *keypair.Full) (initiator Participant, responder Participant) {
```

Comment:
This is a bit of a side change but I noticed we were passing around the `client` but using the global `networkPassphrase`, so I stopped passing the client around for consistency.